app/scripts/controllers/exportImportModal.js

```diff
@@ -0,0 +1,182 @@
+'use strict';
+
+angular.module('openhimConsoleApp')
+  .controller('ExportImportModalCtrl', function ($scope, $modalInstance, $modal, $timeout, Api, Alerting, data) {
+
+    /***************************************************/
+    /**         Initial page load functions           **/
+    /***************************************************/
+
+    $scope.outcome = {
+      selectedAll: false
+    };
+    $scope.ngErrors = {};
+    $scope.conflicts = [];
```

Comment:
I think by default every conflict should have an action of ignore so that the radio button is set.